### PR TITLE
MULE-18805: onError Nesteable Component model fail in lazy initialisation

### PR DESCRIPTION
**Pull request by mbuchwald**

### CODE REVIEW

**Comment by mbuchwald** [comment]
--validate

File: modules/spring-config/src/main/java/org/mule/runtime/config/internal/model/ComponentModel.java [hunk 1]
```diff
@@ -29,8 +29,7 @@
 import static org.mule.runtime.config.internal.model.MetadataTypeModelAdapter.createParameterizedTypeModelAdapter;
 import static org.mule.runtime.core.api.util.StringUtils.trim;
 import static org.mule.runtime.extension.api.util.ExtensionMetadataTypeUtils.isMap;
-import static org.mule.runtime.internal.dsl.DslConstants.KEY_ATTRIBUTE_NAME;
-import static org.mule.runtime.internal.dsl.DslConstants.VALUE_ATTRIBUTE_NAME;
+import static org.mule.runtime.internal.dsl.DslConstants.*;
```
Comment (elrodro83) [inline modules/spring-config/src/main/java/org/mule/runtime/config/internal/model/ComponentModel.java:32]: no `*`

**Review by elrodro83 — CHANGES_REQUESTED** [review]

**Review by elrodro83 — APPROVED** [review]

**Comment by mbuchwald** [comment]
--verify

**Comment by mbuchwald** [comment]
--validate

**Comment by mbuchwald** [comment]
--validate

**Comment by mbuchwald** [comment]
--validate

**Comment by mbuchwald** [comment]
--validate

**Review by elrodro83 — APPROVED** [review]

**Comment by mbuchwald** [comment]
--validate

**Pull request merged**